mne/io/bti/bti.py

```diff
@@ -1143,7 +1143,8 @@ def by_name(x):
         data = _read_data(bti_info)
         assert len(data) == len(info['ch_names'])
         self._projector_hashes = [None]
-        super(RawBTi, self).__init__(info, data, verbose=verbose)
+        super(RawBTi, self).__init__(
+            info, data, filenames=[pdf_fname], verbose=verbose)
```

Comment:
See comments in code, but that would go in `_raw_extras`, which is supposed to be a list the same length as the number of underlying raw files if it's used. You can make whatever you want as entries in the list, `RawFIF` stores one FIF tag directory per file, EDF stores a `dict` and I think KIT does, too